dch/public/app/components/watch-form/watch-form.html

```diff
@@ -0,0 +1,37 @@
+<div class="container" style='align:left'>
+
+<form name = "myWatchForm" ng-submit="submitForm(selectedWatch.id, selectedWatch.startTime, selectedWatch.interval)">
+	<label for="watchSelect">Select a watch ID:</label>
+
+ 	<select name="watchSelect" ng-model="selectedWatch.id">
+  		<option ng-repeat="watch in watches | unique: 'WATCH_ID'">
+        {{watch.WATCH_ID}}
+      </option>
+  </select>
+
+  	<div ng-show="watchSelected">
+  		<label for="timeSelect">Pick start time:</label>
+
+  		<select name="timeSelect" ng-model="selectedWatch.startTime">
+        <option ng-repeat="time in times">
+          {{time}}
+  		</select>
+  	</div>
+
+    <div ng-show="timeSelected">
+      <label for="intervalSelect">Pick a time interval:</label>
+
+      <select name="intervalSelect" ng-model="selectedWatch.interval">
+        <option value="5">Every 5 minutes</option>
+        <option value="10">Every 10 minutes</option>
+        <option value="30">Every 30 minutes</option>
+      </select>
+    </div>
+ 
+	<div>
+		<button type="submit" class="btn btn-success" >Get Data</button>
+	</div>
+
+</form>
+
+</div>
```

Comment:
Be sure the tags match. I see a few that aren't aligned. (Ideally, two space indentaiton)